src/controller/config.js

```diff
@@ -2,34 +2,34 @@ const myImage = document.querySelector('#bodyPart');
 const bodyTagModalElement = document.querySelector("#body_tag_modal")
 const bodyTagBtn = document.querySelector('#tag_btn')
 const bodyTagTxt = document.querySelector("#body_tag")
-const bodyPartModalElement = document.querySelector("#body_part_modal")
-const bodyPartBtn = document.querySelector('#body_part_btn')
-const bodyPartTxt = document.querySelector("#body_part")
 const imgCanvas = document.querySelector("#img_canvas")
 const dropArea = document.querySelector('#body_part_area');
-const newBodyPartBtn = document.querySelector("#new_body_part")
-const saveChecklistBtn = document.querySelector("#save_checklist")
-const checklistTitleTxt = document.querySelector("#checklist_title")
+const saveBodyPartBtn = document.querySelector("#save_body_part")
+const bodyPartTitleTxt = document.querySelector("#body_part_title")
 const title = document.querySelector('title')
+const bodyTagTitle = document.querySelector("#body_tag_title")
 const bodyTagModal = new bootstrap.Modal(bodyTagModalElement)
-const bodyPartModal = new bootstrap.Modal(bodyPartModalElement)
 let noImgPath = '../images/no_image.jpeg'
 let coordinatesMap = {}
-let isBodyPartSaved = false
+let isEdit = false
 let bodyPartId = null
 let checklistId = null
-const bodyPartMap = {}
-const checklistMap = {}
+let bodyTagId = null
 let currCoordinates = null
 
 window.addEventListener('load', (e) => {
     setNoImage()
-    let editBodyPartList = window.api.getEditBodyPart()
-    if(editBodyPartList){
-        let editBodyPart = editBodyPartList[0]
-        bodyPartId = editBodyPartList[1]
-        window.api.clearEditBodyPart()
-        setupEditBodyPart(editBodyPart, bodyPartId)
+    bodyPartId = window.api.getEditBodyPart()
+    checklistId = window.api.getCurrChecklist()
+    if(checklistId){
+        if(bodyPartId){
```

Comment:
if we have a body part it means we are editing a body part so set the flag and set up for editing